doc/source/analytics/log_level.md

```diff
@@ -1,21 +1,33 @@
-# Seldon Core Analytics
+# Logging and log level
 
 Seldon core metrics containers are able to provide different log levels. 
+By default the containers come out of the box with `WARNING` as default log
+level (or the equivalent less verbose settings).
 
-By default the containers come out of the box with WARNING as default log level.
+The verbosity level can be increased to also log `DEBUG` and `INFO` messages by
+following these instructions.
 
-This can be changed into DEBUG, INFO, WARNING and ERROR by following the following instructions.
+## Log level in Python inference servers
 
-## Setting the environment variable
+.. Note:: 
```

Comment:
Just built a doc page to look at it.
Very nice add-on to use `Note` option. Make it good impression when viewing the page.

Reply:
Yeah, they look pretty good! I'd already used them on a couple other pages. They help to highlight information. 

Although it's a bit painful because you have to use RST inside them.